collections/articles.coffee

```diff
@@ -44,7 +44,8 @@ if Meteor.isServer
 
 Meteor.methods
   addEventSource: (source) -> #eventId, url, publishDate, publishDateTZ
-    if source.url.length
+    user = Meteor.user()
+    if Roles.userIsInRole(user._id, ['admin']) and source.url.length
```

Comment:
I see now that this method is both server-side and client-side, so if somebody calls that method via JS console while not being authorized, it'll trigger an exception on the server.
Might be out of scope of this PR, but I'd add a check between L47 and L48 to make sure the user is not null.